<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskManager</class>
 <widget class="QWidget" name="TaskManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="to_do_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">   QLabel {
       background-color: rgb(205, 161, 255);
       border: 2px solid black;
       color: black;
   }
  
</string>
   </property>
   <property name="text">
    <string>To Do</string>
   </property>
  </widget>
  <widget class="QLabel" name="doing_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">   QLabel {
       background-color: rgb(119, 180, 255);
       border: 2px solid black;
       color: black;
   }
  </string>
   </property>
   <property name="text">
    <string>Doing</string>
   </property>
  </widget>
  <widget class="QLabel" name="done_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">   QLabel {
	background-color: rgb(74, 244, 36);
border: 2px solid black;
       color: black;
   }
 </string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QListWidget" name="to_do_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>351</width>
     <height>711</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">   QListWidget {
       border: 2px solid black;
   }
   
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="doing_list">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>351</width>
     <height>711</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">   QListWidget {
       border: 2px solid black;
   }
   
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="done_list">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>351</width>
     <height>711</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">   QListWidget {
       border: 2px solid black;
   }
   
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_to_do_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>700</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">   QPushButton {
       border-radius: 25px;
       background-color: rgb(37, 208, 255);
       border: 1,5px solid black;
   }

   
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_doing_btn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>700</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">   QPushButton {
       border-radius: 25px;
       background-color: rgb(37, 208, 255);
       border: 1,5px solid black;
   }

   
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_done_btn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>700</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">   QPushButton {
       border-radius: 25px;
       background-color: rgb(37, 208, 255);
       border: 1,5px solid black;
   }

   
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
